docs/reference/private_5g_demo_env.md

```diff
@@ -0,0 +1,22 @@
+# Private 5G Demo Environment
+
+Charmed OAI RAN can be used to demonstrate private 5G networks. This document provides an overview of the components that make up the end-to-end demo environment.
+
+The demo environment described here is intended to be as easy to set up as possible, while still providing a realistic demonstration of a private 5G network.
+
+```{image} ../images/demo_env.svg
+:alt: Demo Environment
+:align: center
+```
+
+## Components
+
+- **Host**: A computer with:
+  - 20-core CPU
+  - 16GB RAM
+  - 2 Network Interfaces
```

Comment:
Are you certain about the CPU and memory requirements? I think we might be able to get away with a lot less cores with a recent desktop CPU. Regarding RAM, I think it looks a bit on the small side.

Reply:
I think you're right about the CPU usage, I'll bring it down to 8.

As for the Memory, my demo setup with both the ran and core are using 12GB at the moment.

Reply:
Ok, perfect!